app/src/pages/project/TracesTable.tsx

```diff
@@ -229,8 +253,33 @@ export function TracesTable(props: TracesTableProps) {
         ...rootSpan.descendants.edges.map(({ node }) => node),
       ]);
       // Unwrap the root span from the span tree and return it
-      const [root] = spanTreeToNestedSpanTableRows(spanTree);
-      return root;
+      const [root] = spanTreeToNestedSpanTableRows({
+        children: spanTree,
+      });
+      type SpanRowType = typeof root & IAdditionalSpansRow;
+      // check if there are more spans in the tree than is loaded
+      const numSpansNotLoaded = rootSpan.trace.numSpans - NUM_DESCENDANTS - 1;
+      if (numSpansNotLoaded > 0) {
+        root.children = [
+          ...root.children,
+          // We add a dummy span here to indicate that there are more spans in the tree
+          {
+            ...root,
+            // Indicate that this is an additional row, not a span
+            __additionalRow: true,
+            name: `+ ${numSpansNotLoaded} more span${numSpansNotLoaded > 1 ? "s" : ""}`,
+            id: `additional-${root.id}`,
+            // Clear out the span info
+            input: { value: "" },
+            output: { value: "" },
+            metadata: null,
+            spanAnnotations: [],
+            documentRetrievalMetrics: [],
+            children: [],
+          } as SpanRowType,
+        ];
+      }
+      return root as SpanRowType;
```

Comment:
Same here if it doesn't cause a bunch of extra type debugging

Reply:
this one I can change. Good to know

Reply:
actually know - causes issues all the way down